ui/redux/actions/file.js

```diff
@@ -17,23 +14,11 @@ import { doToast } from 'redux/actions/notifications';
 import { selectBalance } from 'redux/selectors/wallet';
 import { makeSelectFileInfoForUri, selectOutpointFetchingForUri } from 'redux/selectors/file_info';
 import { isStreamPlaceholderClaim } from 'util/claim';
+import { getStripeEnvironment } from 'util/stripe';
+const stripeEnvironment = getStripeEnvironment();
 
 type Dispatch = (action: any) => any;
 type GetState = () => { claims: any, file: FileState, content: any, user: UserState };
-export function doOpenFileInFolder(path: string) {
-  return () => {
-    shell.showItemInFolder(path);
-  };
-}
```

Comment:
If not mistaken, by removing the function above, `openFolder` points to nothing:
```
const perform = (dispatch) => ({
  openFolder: (path) => dispatch(doOpenFileInFolder(path)),
});
```
While `openFolder` itself isn't used in `FileDetails` because if `!SIMPLE_SITE`, this partial removal leaves the code in a confusing limbo state, which requires git to figure out what's wrong.

Either put it back in and leave alone, or complete the removal in the jsx.  I vote the former since (1) less diffs (2) I heard we potentially will have a Desktop app again, so this saves up the need to put it back in?